components/lsof/Makefile

```diff
@@ -0,0 +1,48 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"). You may
+# only use this file in accordance with the terms of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source. A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2016 Jim Klimov. All rights reserved.
+#
+
+include ../../make-rules/shared-macros.mk
+
+COMPONENT_NAME=		lsof
+COMPONENT_VERSION=	4.89
+COMPONENT_PROJECT_URL=	http://freecode.com/projects/util-linux
+COMPONENT_SUMMARY=	LSOF - lister of opened files
+COMPONENT_SRC=		$(COMPONENT_NAME)_$(COMPONENT_VERSION)
+COMPONENT_ARCHIVE=	$(COMPONENT_SRC).tar.bz2
+# Theoretical upstream is ftp://lsof.itap.purdue.edu/pub/tools/unix/lsof
+# but it is not available at the time of this writing
+COMPONENT_ARCHIVE_URL=	ftp://sunsite.ualberta.ca/pub/Mirror/$(COMPONENT_NAME)/$(COMPONENT_ARCHIVE)
+COMPONENT_ARCHIVE_HASH=	\
+    sha256:81ac2fc5fdc944793baf41a14002b6deb5a29096b387744e28f8c30a360a3718
+
+include ../../make-rules/prep.mk
+include ../../make-rules/configure.mk
+include ../../make-rules/ips.mk
+
+COMPONENT_PREP_ACTION = \
```

Comment:
Per se CONFIGURE_SCRIPT does not cut it, unless I uglily override some
other variable to effect `yes ''` to accept the defaults. At least, this
commit point does build 32/64 bit bins so users can install them, and IMHO
improvements like finding real ways to pass build-settings explicitly can
be subject of another PR... at least I won't have time in nearby days to
pursue this.

I will look if there is a quick conversion to justmake in the time I have
today though. But if not - i'd suggest to leave it for another PR too.

--
Typos courtesy of GMail on my Samsung Android
18.01.2016 10:05 пользователь "Alexander Pyhalov" <notifications@github.com>
написал:

> In components/lsof/Makefile
> <https://github.com/OpenIndiana/oi-userland/pull/1643#discussion_r49973954>
> :
>
> > +COMPONENT_VERSION=	4.89
> > +COMPONENT_PROJECT_URL=	http://freecode.com/projects/util-linux
> > +COMPONENT_SUMMARY=	LSOF - lister of opened files
> > +COMPONENT_SRC=		$(COMPONENT_NAME)_$(COMPONENT_VERSION)
> > +COMPONENT_ARCHIVE=	$(COMPONENT_SRC).tar.bz2
> > +# Theoretical upstream is ftp://lsof.itap.purdue.edu/pub/tools/unix/lsof
> > +# but it is not available at the time of this writing
> > +COMPONENT_ARCHIVE_URL=	ftp://sunsite.ualberta.ca/pub/Mirror/$(COMPONENT_NAME)/$(COMPONENT_ARCHIVE)
> > +COMPONENT_ARCHIVE_HASH=	\
> > +    sha256:81ac2fc5fdc944793baf41a14002b6deb5a29096b387744e28f8c30a360a3718
> > +
> > +include ../../make-rules/prep.mk
> > +include ../../make-rules/configure.mk
> > +include ../../make-rules/ips.mk
> > +
> > +COMPONENT_PREP_ACTION = \
>
> Looks a bit ugly. Perhaps, this can be done with
> CONFIGURE_SCRIPT=Configure or using justbuild.mk ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/OpenIndiana/oi-userland/pull/1643/files#r49973954>.
>